Devices/cyd-3248s035c/devicetree.yaml

```
@@ -0,0 +1,3 @@
+dependencies:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)